x/wasm/simulation/params.go

```diff
@@ -0,0 +1,54 @@
+package simulation
+
+import (
+	"fmt"
+	"math/rand"
+
+	"github.com/CosmWasm/wasmd/x/wasm/internal/types"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+)
+
+func ParamChanges(r *rand.Rand) []simtypes.ParamChange {
+	return []simtypes.ParamChange{
+		simulation.NewSimParamChange(types.ModuleName, string(types.ParamStoreKeyUploadAccess),
+			func(r *rand.Rand) string {
+				param := RandomizeAccessConfig(r)
+				return fmt.Sprintf(`{"code_upload_access":"%s"}`, param.String())
```

Comment:
You need a full json object here
```suggestion
				jsonBz, _ := json.Marshal(&param)
				return string(jsonBz)
```